quarto/table-templates/template-table_22.qmd

```diff
@@ -0,0 +1,72 @@
+---
+title: FDA Table 22
+subtitle: Overview of Adverse Events1 by Demographic Subgroup, Safety Population, Pooled Analysis (or Trial X)
+format: html
+---
+
+::: panel-tabset
+## Spec. Screenshot
+
+![](../assets/images/screenshots/table_22.png){fig-align="center"}
+
+## rtables/tern Table
+
+```{r tbl, message=FALSE, warning=FALSE}
+# Load Libraries & Data
+library(dplyr)
+library(scda)
+library(falcon)
+
+adsl <- scda::synthetic_cdisc_dataset("rcd_2022_10_13", "adsl") %>%
+  mutate(AGEGR1 = as.factor(case_when(
+    AGE >= 17 & AGE < 65 ~ ">=17 to <65",
+    AGE >= 65 ~ ">=65",
+    AGE >= 65 & AGE < 75 ~ ">=65 to <75",
+    AGE >= 75 ~ ">=75"
+  )) %>% formatters::with_label("Age Group, years")) %>%
+  formatters::var_relabel(
+    AGE = "Age, years"
+  )
+
+adae <- scda::synthetic_cdisc_dataset("rcd_2022_10_13", "adae")
+
+df <- left_join(adsl, adae, by = intersect(names(adsl), names(adae)))
+
+# Output Table
+make_table_22(df = df, alt_counts_df = adsl, denom = "N_s")
+```
+
+## rtables/tern Table Setup
+
+```{r tbl, eval=FALSE, echo=TRUE}
+```
+
+## Function Details
+
+### `make_table_22()`
+
+------------------------------------------------------------------------
+
+Required variables:
+
+-   **`df`**: `USUBJID`, and the variables specified by `arm_var` and `saffl_var`.
+-   **`alt_counts_df`** (if specified): `USUBJID` and the variables specified by `arm_var` and `saffl_var`.
+
+`make_table_22()` inherits from `falcon::a_count_occurrences_ser_ae()` function.
+
+| Argument         | Description                                                                                                                                                                                                                                            | Default                                                  |
```

Comment:
1. I suggest we include the "treatment-emergent flag" in the argument, similar to "saffl_var". Note: To my knowledge, we haven't addressed the treatment-emergent flag yet. It might be beneficial to discuss this in our meeting. 
2. description of "denom" is not appropriate for this table. 'n' is not the total number of "Any SAE". Please update.